test/credis_test.py

```diff
@@ -1,14 +1,18 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+from __future__ import absolute_import, division, print_function
 
 import os
-import redis
 import unittest
 
+import redis
```

Comment:
please revert, let's keep these in alphabetical order

Reply:
```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Ok sounds good.